Weather/Services/UserLocations/RealmUserLocationsService.swift

```diff
@@ -0,0 +1,85 @@
+import Foundation
+import RealmSwift
+
+
+class RealmWeatherLocation: Object {
+    dynamic var locationId: String = ""
+    dynamic var name: String = ""
+    dynamic var region: String = ""
+    dynamic var country: String = ""
+    var latitude = RealmOptional<Double>()
+    var longitude = RealmOptional<Double>()
+}
+
+
+class RealmUserLocationsService: UserLocationsService {
```

Comment:
Can we change the protocol name to something like:

LocationStoreService

as otherwise it looks like something that relates to the actual user location vs a store for locations that user choose :)